### ISSUE TITLE
[4.0.0-beta.5] Bug: Infinite spinner when creating a new wallet, CSP PR has broken login

### ISSUE THREAD

**Issue by timstackblock** [blockstack/stacks-wallet#312]
During the creation of a new wallet once the user enters a password they get an infinite spinner and are stuck on the screen below

<img width="1007" alt="Screen Shot 2020-10-30 at 10 26 17 AM" src="https://user-images.githubusercontent.com/49165468/97716825-7844e980-1a9a-11eb-94b3-194a05ad9e44.png">

Bug found testing Stacks Wallet build ff021be, Thu Oct 29 12:50:07 UTC 2020.

**Comment by kyranjamie** [comment]
Good find, @timstackblock —

I've ran into some issues with this PR where things are only breaking in production mode (not even with `DEBUG_PROD` enabled), which is often the last thing I test. I spent quite a lot of time debugging this on Friday. Electron offers a communication mechanism to communicate between the renderer (browser) and main (node) threads. See [ipcMain docs on Electron](https://www.electronjs.org/docs/api/ipc-main). I haven't found yet what it is enabled in prod that breaks this.
